app/assets/javascripts/backbone/views/sortable_form_items_view.js.coffee

```diff
@@ -0,0 +1,6 @@
+class ELMO.Views.SortableFormItemsView extends Backbone.View
+
+  initialize: ->
+    $('.form-items-list, .form-group ol').sortable({connectWith: ".item-list"})
```

Comment:
Just realized you're not using nestedSortable here. Does sortable work with nesting? Check this out: http://mjsarfatti.com/sandbox/nestedSortable/